cli/js/files.ts

```diff
@@ -92,8 +116,14 @@ export function createSync(path: string): File {
  *
  * Requires `allow-read` and `allow-write` permissions.
  */
-export function create(path: string): Promise<File> {
-  return open(path, "w+");
+export function create(path: string, mode?: number): Promise<File> {
```

Comment:
create should not have a mode argument: https://golang.org/pkg/os/#Create

It should default to 0666 before umask.